pom.xml

```diff
@@ -288,6 +288,9 @@ under the License.
                         </licenseFamily>
                     </licenseFamilies>
                     <excludes>
+                        <!-- IDE files. -->
+                        <exclude>**/*.iml</exclude>
+                        <exclude>**/.idea/**</exclude>
```

Comment:
Are they necessary? IMO these two patterns should be redundant while there is `**/.*/**` below.

Reply:
Yes,not need to add exclude.